ui/store/actions.ts

```diff
@@ -3447,10 +3462,14 @@ export function detectNfts(): ThunkAction<
   AnyAction
 > {
   return async (dispatch: MetaMaskReduxDispatch) => {
-    dispatch(showLoadingIndication());
+    dispatch(showNftStillFetchingIndication());
     log.debug(`background.detectNfts`);
-    await submitRequestToBackground('detectNfts');
-    dispatch(hideLoadingIndication());
+    try {
+      await submitRequestToBackground('detectNfts');
+      dispatch(hideNftStillFetchingIndication());
```

Comment:
This line seems unnecessary since it's already in the finally block?

Reply:
Yup thnx for catching that